<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordMoviesView</class>
 <widget class="QWidget" name="RecordMoviesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>441</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>File:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>fileText</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileText"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>titleText</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleText"/>
     </item>
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
       <property name="buddy">
        <cstring>typeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeComboBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hunt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Explore</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Player Kill</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>War</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Party</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="buddy">
      <cstring>descriptionText</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionText"/>
   </item>
   <item>
    <widget class="QGroupBox" name="recordOptionsGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Record Options</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="hideVipCheckBox">
          <property name="text">
           <string>Hide viplist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hidePrivateCheckbox">
          <property name="text">
           <string>Hide private messages</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="hideItemsCheckBox">
          <property name="text">
           <string>Hide items</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hideSkillsCheckBox">
          <property name="text">
           <string>Hide skills</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="recordStatusGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Record Status</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="statusLabelValue">
            <property name="styleSheet">
             <string>color: red; font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Idle</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="timeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Time Recorded:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timeLabelValue">
            <property name="text">
             <string notr="true">00:00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPushButton" name="startRecordingButton">
         <property name="text">
          <string>Start Recording</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopRecordingButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop Recording</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
